Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -92,6 +92,17 @@ public override object DeserializeObject(object value, Type type)
                 {
                     if (ReflectionUtils.GetTypeInfo(type).IsEnum)
                     {
+                        //first try to get all custom attributes
```

Comment:
It's very unlikely that an enum is going to have a huge number of elements. And we only need to iterate once and cache the result. An enum change would require recompilation. So I'm happy with ENUMeration. :stuck_out_tongue: